main.py

```diff
@@ -22,9 +23,6 @@
     # timeout=120,
 )
 def generate_response(system_prompt, user_prompt, *args):
-    import openai
-    import tiktoken
```

Comment:
can you undo this? it is intentional for modal https://modal.com/docs/guide/custom-container#importing-python-packages

Reply:
maybe add a comment for future readers

Reply:
Already on it. Thanks for your feedback.